examples/security/README.md

```diff
@@ -0,0 +1,48 @@
+Users provisioning
+===
+
```

Comment:
Add a note highlighting that once the security has been enabled and GraphDB provisioned, changes to the users in values.yaml or users.js won't take effect.